portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +28,7 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
+import java.util.Iterator;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```

Comment:
Just noticed, should we remove this TODO/comment now that it's mostly implemented?